test/jira_client_test.rb

```diff
@@ -0,0 +1,21 @@
+# frozen_string_literal: true
+
+require "test_helper"
+require "otis/jira_client"
+
+module Otis
+  class JiraClientTest < ActiveSupport::TestCase
+    test "make sure we're read-only" do
+      assert Otis.config.jira.readonly
+    end
+
+    test "comment template test" do
+      comment = Otis::JiraClient.comment template: :registration_sent, user: "nobody@default.invalid"
```

Comment:
It doesn't look like this actually connects to anything, so do we even need the null jira object?

Reply:
I've placed the comment templates in the so-called Jira client so they're accessible to both the registration controller and the finalize controller, and semantically they're "stuff we send to Jira". There's no supposition of any sort of HTTP client, it's more of a utility method.

Reply:
:+1: